src/data_handler.py

```diff
@@ -0,0 +1,160 @@
+#import BoardInterface
+
+from enum import Enum
+
+class Sensor():
```

Comment:
Some comments could really be used in the code here - even if just at the class level. For example, for the sensor class, maybe write something along the lines of 
"""
This is an abstract class representing a single sensor on the test stand. Derived classes must overwrite the `_rawToRealValue` method with the specific logic that will turn a raw value into an actual sensor measurement.
"""